src/server/controllers/index.js

```diff
@@ -0,0 +1,84 @@
+const  express = require('express');
+var router = express.Router();
+
+/* Require model */
+var models = require('../models');
+/* Require middlewares functions */
+
+router.get('/', function (req, res) {
```

Comment:
What is the difference between this and code in

https://github.com/itexico/interview-codingchallenge-fsjs/pull/10/files#diff-4611e0745a3eb7683e1e32a9a677fb8bR15
?